LQResultProducer.py

```diff
@@ -4042,8 +4099,28 @@ def SysTable(optimlog, selection_uujj,selection_uvjj,NormalDirectory, weight,sys
 			munu1 = '(((CISV_jet1>0.5426)+(CISV_jet2>0.5426))<1)*(MT_uv>70)*(MT_uv<110)*(1-((CISV_jet1>0.5426)*(0.887973*((1.+(0.0523821*Pt_jet1))/(1.+(0.0460876*Pt_jet1)))-(0.02891194075345993*(Pt_jet1>50)*(Pt_jet1<70)+0.028121808543801308*(Pt_jet1>70)*(Pt_jet1<100)+0.027028990909457207*(Pt_jet1>100)*(Pt_jet1<140)+0.027206243947148323*(Pt_jet1>140)*(Pt_jet1<200)+0.033642303198575974*(Pt_jet1>200)*(Pt_jet1<300)+0.04273652657866478*(Pt_jet1>300)*(Pt_jet1<600)+0.054665762931108475*(Pt_jet1>600)))))*(1-((CISV_jet2>0.5426)*(0.887973*((1.+(0.0523821*Pt_jet2))/(1.+(0.0460876*Pt_jet2)))-(0.02891194075345993*(Pt_jet2>50)*(Pt_jet2<70)+0.028121808543801308*(Pt_jet2>70)*(Pt_jet2<100)+0.027028990909457207*(Pt_jet2>100)*(Pt_jet2<140)+0.027206243947148323*(Pt_jet2>140)*(Pt_jet2<200)+0.033642303198575974*(Pt_jet2>200)*(Pt_jet2<300)+0.04273652657866478*(Pt_jet2>300)*(Pt_jet2<600)+0.054665762931108475*(Pt_jet2>600)))))'
 			munu2 = '(((CISV_jet1>0.8484)+(CISV_jet2>0.8484))>=1)*(MT_uv>70)*(MT_uv<110)*(1-(1-((CISV_jet1>0.8484)*(0.561694*((1.+(0.31439*Pt_jet1))/(1.+(0.17756*Pt_jet1)))-(0.03711806982755661*(Pt_jet1>50)*(Pt_jet1<70)+0.036822021007537842*(Pt_jet1>70)*(Pt_jet1<100)+0.034397732466459274*(Pt_jet1>100)*(Pt_jet1<140)+0.0362386554479599*(Pt_jet1>140)*(Pt_jet1<200)+0.044985830783843994*(Pt_jet1>200)*(Pt_jet1<300)+0.064243391156196594*(Pt_jet1>300)*(Pt_jet1<600)+0.097131341695785522*(Pt_jet1>600)))))*(1-((CISV_jet2>0.8484)*(0.561694*((1.+(0.31439*Pt_jet2))/(1.+(0.17756*Pt_jet2)))-(0.03711806982755661*(Pt_jet1>50)*(Pt_jet1<70)+0.036822021007537842*(Pt_jet1>70)*(Pt_jet1<100)+0.034397732466459274*(Pt_jet1>100)*(Pt_jet1<140)+0.0362386554479599*(Pt_jet1>140)*(Pt_jet1<200)+0.044985830783843994*(Pt_jet1>200)*(Pt_jet1<300)+0.064243391156196594*(Pt_jet1>300)*(Pt_jet1<600)+0.097131341695785522*(Pt_jet1>600))))))'
 
-	[[Rz_uujj,Rz_uujj_err],[Rtt_uujj,Rtt_uujj_err]] = GetMuMuScaleFactors( selection_uujj, NormalDirectory, '(M_uu>80)*(M_uu<100)', '(M_uu>100)',0,0)
-	[[Rw_uvjj,Rw_uvjj_err],[Rtt_uvjj,Rtt_uvjj_err]] = GetMuNuScaleFactors( selection_uvjj, NormalDirectory, munu1,munu2,0)
+	#[[Rz_uujj,Rz_uujj_err],[Rtt_uujj,Rtt_uujj_err]] = GetMuMuScaleFactors( selection_uujj, NormalDirectory, '(M_uu>80)*(M_uu<100)', '(M_uu>100)*(M_uu<250)',0,0)
+	#if year == '2016':
+	#	[[Rz_uujj,Rz_uujj_err],[Rtt_uujj,Rtt_uujj_err]] = [[1.019,0.017],[1.134,0.013]] #2016 stock NanoAODv7 with 1 btag
+	#else:
+	#	[[Rz_uujj,Rz_uujj_err],[Rtt_uujj,Rtt_uujj_err]] = GetMuMuScaleFactors( NormalWeightMuMu+'*'+preselectionmumu, NormalDirectory, '(M_uu>80)*(M_uu<100)', '(M_uu>100)*(M_uu<250)',0,0)
+	#[[Rw_uvjj,Rw_uvjj_err],[Rtt_uvjj,Rtt_uvjj_err]] = [[1.0,0.0],[1.0,0.0]]
+	#[[Rw_uvjj,Rw_uvjj_err],[Rtt_uvjj,Rtt_uvjj_err]] = GetMuNuScaleFactors( selection_uvjj, NormalDirectory, munu1,munu2,0)
+
+	# SFs with updated integrated luminosities
+	if year == '2016':
```

Comment:
you can't hardcode these.....they will change with the systematic variations. You have to let them recalculate. But that also takes out the yearly differences, so the code can be simpler, I think just reverted to the original, with the correct TTBar normalization control region (100-250)